### PR TITLE
add GitHub Actions build status

### PR DESCRIPTION
**Pull request by JacobCoffee**
Add CI status for:
- Docs Building
- GH Pages Building
- Release Building
- CI Status

# PR Checklist

- [x] Have you followed the guidelines in `CONTRIBUTING.rst`?
- [ ] Have you got 100% test coverage on new code?
- [x] Have you updated the prose documentation?
- [x] Have you updated the reference documentation?

### CODE REVIEW

**Review by provinzkraut — COMMENTED** [review]
Not sure about the docs and GH pages badges. Seems like it just doesn't really add anything of value?
The release badge, maybe we can add a PyPi badge instead of the current version?

**Comment by JacobCoffee** [comment]
> Not sure about the docs and GH pages badges. Seems like it just doesn't really add anything of value? The release badge, maybe we can add a PyPi badge instead of the current version?


On the docs, I was thinking it may be helpful to know if docs builds fail at a glance but maybe not?
I have replaced the others with a PyPI version # and a PyPI downloads / month counter. 

Thoughts?

**Review by Goldziher — APPROVED** [review]

**Comment by Jarmos-san** [comment]
> Seems like it just doesn't really add anything of value?

README bling :laughing:

**Comment by provinzkraut** [comment]
> On the docs, I was thinking it may be helpful to know if docs builds fail at a glance but maybe not? I have replaced the others with a PyPI version # and a PyPI downloads / month counter.
> 
> Thoughts?

Well my thoughts are the same as before and I agree with @Jarmos-san: It's bling and I don't really like bling if it doesn't add anything of value. PyPi version + build status are useful, the rest not so much.

But if others feel differently about this I'm willing to have my mind changed.

File: README.md [hunk 1]
```diff
@@ -4,11 +4,14 @@
 
 <div align="center">
 
+[![ci](https://github.com/starlite-api/starlite/actions/workflows/ci.yaml/badge.svg)](https://github.com/starlite-api/starlite/actions/workflows/ci.yaml)
+[![PyPI version](https://badge.fury.io/py/starlite.svg)](https://badge.fury.io/py/starlite)
+[![PyPI - Downloads](https://img.shields.io/pypi/dm/starlite)](https://pypistats.org/packages/starlite)
```
Comment (provinzkraut) [inline README.md:9]:
```suggestion
```

**Pull request merged**